CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+title: Code of Conduct
+nav_exclude: true
+permalink: /code-of-conduct
+---
```

Comment:
What will these metadata be used for?
Is this preparation work to publish these pages as GitHub Pages or similar?

Reply:
That is correct. However, I decided to remove it as it is not used now = confusing.